src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -230,48 +230,48 @@ LedgerConsensusImp::LedgerConsensusImp (
         LocalTxs& localtx,
         LedgerMaster& ledgerMaster,
         FeeVote& feeVote)
-    : app_ (app)
```

Comment:
FWIW, Sutter and Alexandrescu argue against what they call "underhanded names" (ones that begin with an underscore or that contain a double underscore) in item 0 of _C++_ _Coding_ _Standards_.